Lib/defconQt/glyphCollectionView.py

```diff
@@ -12,7 +12,6 @@
 cellHeaderBaseColor = QColor(230, 230, 230)
 cellHeaderLineColor = QColor(220, 220, 220)
 cellHeaderHighlightLineColor = QColor(240, 240, 240)
-cellSelectionColor = QColor.fromRgbF(.2, .3, .7, .15)
```

Comment:
this is why the tests are failing. The `cellSelectionColor` variable is imported in `fontInfo_test` module, but since you have moved it from the module's global scope the import fails and the test cannot be loaded.